### PR TITLE
Fixes #19 Keep an eye on Redis Servers

### LINKED ISSUE #19: Monitor redis

### PR DESCRIPTION
**Pull request by roger-russel**
This merge will allow Beholder Keep on eye on Redis Servers.
At moment it will be only connection without authentication.

### CODE REVIEW

File: src/Eyes/Db/Redis/Phpredis.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Db\Redis;
+
+use Redis;
+use Exception;
+use BeholderWebClient\Eyes\Db\Redis\RedisStatus as Status;
+use BeholderWebClient\Eyes\Db\AbstractAdapter;
+
+class Phpredis extends AbstractAdapter {
```
Comment (alanyoshida) [inline src/Eyes/Db/Redis/Phpredis.php]: CamelCase

File: src/Eyes/Db/Redis/Eye.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        throw new Exception("Driver: {$adapter}, is not implemented", Status::INTERNAL_SERVER_ERROR);
+      }
+
+  }
+
+  protected function autoDetectAdapter(){
+
+    switch(true){
+      case class_exists("Redis"):
+        $this->adapter = new Phpredis($this->conf);
```
Comment (alanyoshida) [inline src/Eyes/Db/Redis/Eye.php]: CamelCase Phpredis

File: src/Eyes/Db/Redis/Phpredis.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  public function testQuery(){
+
+    $mergedArrQuery = $this->getMergedArrayQuery();
+
+    $result = false;
+
+    foreach( $mergedArrQuery as $arrQuery ){
+      foreach( $arrQuery['query'] as $query ) {
+
+          $result = mysql_query($query);
```
Comment (alanyoshida) [inline src/Eyes/Db/Redis/Phpredis.php]: mysql_query ?

File: src/Eyes/Db/Redis/Phpredis.php (hunk 1 above)
Comment (roger-russel) [inline src/Eyes/Db/Redis/Phpredis.php]: Phpredis is the lib name it should not use CamelCase like "Lightpower", or "Lightspeed".
Is not like I'm given this name, I'm olny using it.

File: src/Eyes/Db/Redis/Eye.php (hunk 2 above)
Comment (roger-russel) [inline src/Eyes/Db/Redis/Eye.php]: Phpredis is the lib name it should not use CamelCase like "Lightpower", or "Lightspeed".
Is not like I'm given this name, I'm olny using it.

**Comment by alanyoshida** [comment]
Everything looks ok :+1:

**Comment by alanyoshida** [comment]
Approved !

**Pull request merged**